multimodal/src/autogluon/multimodal/optimization/lit_module.py

```diff
@@ -239,13 +239,13 @@ def training_step(self, batch, batch_idx):
 
     def on_validation_start(self) -> None:
         if self.skip_final_val and self.trainer.should_stop:
-            self.trainer.val_dataloaders = []  # skip the final validation by setting val_dataloaders empty
             self.log(
                 self.validation_metric_name,
                 self.validation_metric,
                 on_step=False,
                 on_epoch=True,
             )
+            return None
```

Comment:
@suzhoum Can you verify the fast build mode still behaves well after this change?

Reply:
more context in the commit message: https://github.com/autogluon/autogluon/pull/3190/commits/18fbb5f095fb39df1d1cdae07d16dccf4615cca8